webapp/channels/src/components/with_tooltip/with_tooltip_new/tooltip_shortcut.test.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {defineMessage} from 'react-intl';
+
+import {renderWithContext, screen} from 'tests/react_testing_utils';
+import * as userAgentUtils from 'utils/user_agent';
+
+import TooltipShortcut from './tooltip_shortcut';
+
+jest.mock('utils/user_agent', () => ({
+    isMac: jest.fn(),
+}));
+
+describe('TooltipShortcut', () => {
+    let isMacMock: jest.Mock;
+
+    beforeEach(() => {
+        isMacMock = jest.mocked(userAgentUtils.isMac);
```

Comment:
Yeah, that's right. It's basically just `(a: unknown) => {return a as jest.Mock;}` with the extra typing to ensure that the mock otherwise behaves the same

Reply:
move it outside